eq/qt/messagePump.cpp

```diff
@@ -17,12 +17,19 @@
 
 #include "messagePump.h"
 
+#include <lunchbox/clock.h>
+#include <lunchbox/sleep.h>
 #include <QEventLoop>
 
 namespace eq
 {
 namespace qt
 {
+namespace
+{
+static lunchbox::a_int32_t _wakeup; // QEventLoop is a singleton...
```

Comment:
imo there is a QEventLoop per thread. Why not make the wakeup a member var of MessagePump?